src/node_v8.cc

```diff
@@ -295,6 +295,154 @@ static void SetHeapStatistics(JSONWriter* writer, Isolate* isolate) {
   writer->json_arrayend();
 }
 
+static v8::Local<v8::Object> ConvertHeapStatsToJSObject(
```

Comment:
Usually in the .cc files, we add `using v8::...` to the top and them just use the v8 types without the namespace.